.github/workflows/ci.yml

```diff
@@ -104,7 +103,7 @@ jobs:
 
       - name: Install Python dependencies
         run: |
-          pip install --upgrade pip setuptools py
+          pip install "pip<20.3"
```

Comment:
I guess we may need to keep `--upgrade` in the `pip install...` command in case `pip` is already present in the base environment?

Reply:
It looks like with the current command it downgrades it correctly: [link](https://github.com/reanahub/reana-commons/pull/244/checks?check_run_id=1649481681#step:5:10)